go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/spire
 
 go 1.19
 
+replace github.com/spiffe/go-spiffe/v2 => github.com/guilhermocc/go-spiffe/v2 v2.0.0-20230308163427-b2b17d68664c
```

Comment:
The commit that inserted this temporary dependency should be dropped after https://github.com/spiffe/go-spiffe/pull/220/files gets merged